en/docs/install-and-setup/install/installing-the-product/installing-api-m-runtime.md

```diff
@@ -22,15 +22,15 @@ You must set your `JAVA_HOME` environment variable to point to the directory whe
 ??? note "On Linux/OS X"
 
     1.  In your home directory, open the BASHRC file (.bash\_profile file on Mac) using editors such as vi, emacs, pico, or mcedit.
-    2.  Assuming you have JDK 1.8.0\_xx in your system, add the following two lines at the bottom of the file, replacing `/usr/java/jdk1.8.0_xx` with the actual directory where the JDK is installed.
+    2.  Assuming you have JDK 11 in your system, add the following two lines at the bottom of the file, replacing `/usr/java/jdk-11.0.x` with the actual directory where the JDK is installed.
 
         ``` java
         On Linux:
-        export JAVA_HOME=/usr/java/jdk1.8.0_xx
+        export JAVA_HOME=/usr/java/jdk-11.0.x
         export PATH=${JAVA_HOME}/bin:${PATH}
              
         On OS X:
-        export JAVA_HOME=/System/Library/Java/JavaVirtualMachines/1.8.0.jdk/Contents/Home
+        export JAVA_HOME=/System/Library/Java/JavaVirtualMachines/11.0.x.jdk/Contents/Home
```

Comment:
This didn't work for me actually. Did you test this?

Reply:
you have to use the installed JDK version.